src/Templates/src/templates/maui-mobile/PageModels/ProjectDetailPageModel.cs

```diff
@@ -0,0 +1,177 @@
+#nullable disable
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using MauiApp._1.Models;
+
+namespace MauiApp._1.PageModels;
+
+public partial class ProjectDetailPageModel : ObservableObject, IQueryAttributable, IProjectTaskPageModel
+{
+	private readonly ProjectRepository _projectRepository;
+	private readonly TaskRepository _taskRepository;
+	private readonly CategoryRepository _categoryRepository;
+	private readonly TagRepository _tagRepository;
+	private readonly ModalErrorHandler _errorHandler;
+
+	private Project _project;
+	[ObservableProperty] private string _name;
+	[ObservableProperty] private string _description;
+	[ObservableProperty] private List<ProjectTask> _tasks;
+	[ObservableProperty] private List<Category> _categories;
```

Comment:
on other files the attribute is above the field, would be good to keep the pattern

Reply:
This should be all fixed now